skbio/io/tests/test_util.py

```diff
@@ -522,6 +522,34 @@ def get_contents(self, file):
             return f.read()
 
 
+class TestReadNamedTemporaryFile(ReadableBinarySourceTests,
+                                 ReadableSourceTest):
+    expected_close = False
+
+    def get_fileobj(self, path):
+        fileobj = tempfile.NamedTemporaryFile()
+        with io.open(path, mode='rb') as fh:
+            fileobj.write(fh.read())
+            fileobj.flush()
+            fileobj.seek(0)
+        return fileobj
+
+
+class TestWriteNamedTemporaryFile(WritableBinarySourceTests,
+                                  WritableSourceTest):
+    expected_close = False
+
+    def get_fileobj(self, path):
+        return tempfile.NamedTemporaryFile()
+
+    def get_contents(self, file):
+        file.flush()
+        with io.open(file.name, mode='rb') as f:
```

Comment:
Calling `.name` on a `NamedTemporaryFile` won't work on Windows. Not blocking since we don't yet support Windows but would be nice to fix here if it's easy.